composition-js/src/merging/merge.ts

```diff
@@ -681,7 +681,8 @@ class Merger {
       } else {
         for (const key of keys) {
           const extension = key.ofExtension() || source.hasAppliedDirective(sourceMetadata.extendsDirective()) ? true : undefined;
-          dest.applyDirective(joinTypeDirective, { graph: name, key: key.arguments().fields, extension });
+          const resolvable = key.arguments().resolvable === false ? false : undefined;
```

Comment:
Same here as earlier, if we just want to copy the value in `key.arguments().resolvable`, we should do so rather than use a ternary operator. `const { resolvable } = args;`